dev/src/DocGenerator/Parser/CodeParser.php

```diff
@@ -427,14 +429,10 @@ private function buildReturns($returns)
     private function handleTypes($types)
     {
         foreach ($types as &$type) {
-            // object is a PHPDoc keyword so it is not capable of detecting the context
-            // https://github.com/phpDocumentor/ReflectionDocBlock/blob/2.0.4/src/phpDocumentor/Reflection/DocBlock/Type/Collection.php#L37
-            if ($type === 'Object') {
```

Comment:
Since we don't have a class called Google\Cloud\Storage\Object anymore, this was dead code.